docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -1,37 +1,33 @@
 # Configuration
-To extract the configuration metadata from an application/library the _component extractor_ will be used. It extracts the list of all the config interfaces defined in a library or application. The output is a json file `component.config.metadata.json` containing an array of all the components configurations (app configs, pages, components).
-It will also generate the component classes and types metadata from an Otter library/application, outputting them in a json file `component.class.metadata.json`.
+The component extractor is used to extract the configuration metadata from an application or a library. It extracts the list of all the config interfaces defined
+in a library or an application. The output is a JSON file `component.config.metadata.json` containing an array of all the component configurations (app configurations, pages, components).
```

Comment:
Not sure to understand the parentheses content.
Is part of the "Component Configurations"? but in this case, not sure we should indicate "app configurations". Or is the different types of configurations? If so a link to describe the different type of configs can help and maybe it would be more readable written in that way: `app, pages and components configurations` without being between parentheses